### PR TITLE
Translation of base_chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

Please review.
I translated base_chapter.html and index.html.

### CODE REVIEW

File: src/templates/ja/2019/base_chapter.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 {% block title %}{{ metadata.get('title') }} | 2019 | HTTP ArchiveによるWeb年鑑{% endblock %}
 
-{% block description %}{{ metadata.get('description',metadata.get('title') + ' chapter of the ' + year + ' Web Almanac probing into the use of ' + metadata.get('description',metadata.get('title')) + ' on the web.') }}{% endblock %}
+{% block description %}{{ metadata.get('description',metadata.get('title') + ' の章 ' + year + ' Web年鑑の使用状況を調べる ' + metadata.get('description',metadata.get('title')) + ' をウェブ上で公開しています。') }}{% endblock %}
```
Comment (bazzadp) [inline src/templates/ja/2019/base_chapter.html:5]: I'm don't think we should translate Web Almanac. That's a brand name, as discussed here: https://github.com/HTTPArchive/almanac.httparchive.org/issues/567#issuecomment-559613733
```suggestion
{% block description %}{{ metadata.get('description',metadata.get('title') + ' の章 ' + year + ' Web Almanacの使用状況を調べる ' + metadata.get('description',metadata.get('title')) + ' をウェブ上で公開しています。') }}{% endblock %}
```

File: src/templates/ja/2019/base_chapter.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
-{% block twitter_image_alt %}Chapter image for the {{ metadata.get('title') }} chapter of the {{ year }} Web Almanac{% endblock %}
+{% block twitter_image_alt %}のチャプター画像 {{ metadata.get('title') }} の章 {{ year }} Web年鑑{% endblock %}
```
Comment (bazzadp) [inline src/templates/ja/2019/base_chapter.html:7]:
```suggestion
{% block twitter_image_alt %}のチャプター画像 {{ metadata.get('title') }} の章 {{ year }} Web Almanac{% endblock %}
```

File: src/templates/ja/2019/index.html [hunk 3]
```diff
@@ -1,32 +1,32 @@
 {% extends "base/2019/index.html" %}
 
-{% block title %}The {{ year }} Web Almanac{% endblock %}
-{% block description %}The Web Almanac is an annual state of the web report combining the expertise of the web community with the data and trends of the HTTP Archive.{% endblock %}
+{% block title %}The {{ year }} Web年鑑{% endblock %}
+{% block description %}Web年鑑は、Webコミュニティの専門知識とHTTPアーカイブのデータやトレンドを組み合わせた、毎年恒例のWeb状態のレポートです。{% endblock %}
```
Comment (bazzadp) [inline src/templates/ja/2019/index.html:4]:
```suggestion
{% block title %}The {{ year }} Web年鑑{% endblock %}
{% block description %}Web Almanacは、Webコミュニティの専門知識とHTTP Archiveーカイブのデータやトレンドを組み合わせた、毎年恒例のWeb状態のレポートです。{% endblock %}
```

File: src/templates/ja/2019/index.html [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
 
-{% block twitter_image_alt %}The {{ year }} Web Almanac{% endblock %}
+{% block twitter_image_alt %}The {{ year }} Web年鑑{% endblock %}
```
Comment (bazzadp) [inline src/templates/ja/2019/index.html:6]: Don't translate Web Almanac. But what about the word "The" at the beginning?

```suggestion
{% block twitter_image_alt %}The {{ year }} Web Almanac{% endblock %}
```

File: src/templates/ja/2019/index.html [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
 
 {% block date_published %}2019-11-04T12:00:00+00:00:00{% endblock %}
 {% block date_modified %}2019-11-04T12:00:00+00:00:00{% endblock %}
 
-{% block breadcrumb_name_1 %}{{ year }} Home{% endblock %}
+{% block breadcrumb_name_1 %}{{ year }} ホーム{% endblock %}
 
-{% block intro_title %}Web Almanac{% endblock %}
-{% block intro_sub_title %}HTTP Archive's annual<br> <b>state of the web</b> report{% endblock %}
+{% block intro_title %}Web年鑑{% endblock %}
```
Comment (bazzadp) [inline src/templates/ja/2019/index.html:13]:
```suggestion
{% block intro_title %}Web Almanac{% endblock %}
```

File: src/templates/ja/2019/index.html [hunk 6]
```diff
@@ -1,32 +1,32 @@
...
 
-{% block methodology_stat_1_title %}Websites Tested{% endblock %}
+{% block methodology_stat_1_title %}テストしたウェブサイト{% endblock %}
 {% block methodology_stat_1 %}5.8M{% endblock %}
-{% block methodology_stat_2_title %}Data Processed{% endblock %}
+{% block methodology_stat_2_title %}処理されたデータ{% endblock %}
 {% block methodology_stat_2 %}20.9 TB{% endblock %}
 {% block methodology_description %}
-Unless otherwise noted, the metrics in all of the 20 chapters of the Web Almanac are sourced from the HTTP Archive dataset. HTTP Archive is a community-run project that has been tracking how the web is built since 2010. Using WebPageTest and Lighthouse under the hood, metadata about nearly 6 million websites are tested monthly and included in a public BigQuery database for analysis. The July 2019 dataset was used as the basis for the Web Almanac's metrics. For more information, see the Methodology page.
+特に断りのない限り、Web年鑑の20章すべてのメトリクスは、HTTP Archiveのデータセットから取得しています。HTTP Archiveは、2010年以来、ウェブの構築方法を追跡してきたコミュニティが運営するプロジェクトです。WebPageTestとLighthouseをアンダーグラウンドに使用して、約600万のWebサイトに関するメタデータが毎月テストされ、分析のために公開されているBigQueryデータベースに含まれています。2019年7月のデータセットは、Web年鑑のメトリクスの基礎として使用されました。詳細については、方法論のページを参照してください。
```
Comment (bazzadp) [inline src/templates/ja/2019/index.html:30]:
```suggestion
特に断りのない限り、Web Almanacの20章すべてのメトリクスは、HTTP Archiveのデータセットから取得しています。HTTP Archiveは、2010年以来、ウェブの構築方法を追跡してきたコミュニティが運営するプロジェクトです。WebPageTestとLighthouseをアンダーグラウンドに使用して、約600万のWebサイトに関するメタデータが毎月テストされ、分析のために公開されているBigQueryデータベースに含まれています。2019年7月のデータセットは、Web Almanacのメトリクスの基礎として使用されました。詳細については、方法論のページを参照してください。
```

File: src/templates/ja/2019/base_chapter.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2019/base_chapter.html:5]: OK, I understand.

But what about the chapters I've already translated (cms.md, http2.md, etc.)?
These chapters have been translated into the "Web年鑑".

File: src/templates/ja/2019/base_chapter.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2019/base_chapter.html:7]: I fixed.

File: src/templates/ja/2019/index.html (hunk 3 above)
Comment (ksakae1216) [reply src/templates/ja/2019/index.html:4]: I fixed.

File: src/templates/ja/2019/index.html (hunk 4 above)
Comment (ksakae1216) [reply src/templates/ja/2019/index.html:6]: I fixed.

File: src/templates/ja/2019/index.html (hunk 5 above)
Comment (ksakae1216) [reply src/templates/ja/2019/index.html:13]: I fixed.

File: src/templates/ja/2019/index.html (hunk 6 above)
Comment (ksakae1216) [reply src/templates/ja/2019/index.html:30]: I fixed.

File: src/templates/ja/2019/base_chapter.html (hunk 1 above)
Comment (bazzadp) [reply src/templates/ja/2019/base_chapter.html:5]: Yeah we should fix those too. Can you open a separate pull request for that?

Comment (bazzadp) [reply src/templates/ja/2019/base_chapter.html:5]: I can do a quick search and replace fairly easily if you want me to?

Comment (ksakae1216) [reply src/templates/ja/2019/base_chapter.html]: Yes, I will create other branch and fix.

**Pull request merged**